### DISCUSSION TITLE
HasMany отключение кнопки создания в режиме resourceMode

### DISCUSSION THREAD

**Discussion by IgorPetrovPRO** [moonshine-software/moonshine#344] [Ideas]
Выводя на странице через HasMany в режиме resourceMode  всегда выводится кнопка создания, не смотря на то, что указано в $activeActions https://disk.yandex.ru/d/hqvztkR-IA5SFQ показалось что не хватает параметра для скрытия и этой кнопки.

**Comment by lee-to** [comment]
Исправили в релизе 1.59.0! Спасибо
